card/lib/card/query.rb

```diff
@@ -70,11 +70,12 @@ class Query
 
     DEFAULT_ORDER_DIRS =  { :update => "desc", :relevance => "desc" }
 
-    attr_reader :statement, :mods, :conditions,
+    attr_reader :statement, :mods, :conditions, :comment,
       :subqueries, :superquery
     attr_accessor :joins, :table_seq, :unjoined, :conditions_on_join
 
-    def initialize statement
+    def initialize statement, comment=nil
+      @comment = comment
```

Comment:
add optional comment to queries.  Search cards send their name as a comment.

Comment shows up in rails logging.